### ISSUE TITLE
SplitButton borders and divider & PrimaryButton Focus not visible in high contrast/Edge

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#10299]
Standard Split button divider is not visible in high contrast mode.
Primary Split button 'split' button border is not visible in high contrast mode

Edge browser.
https://fabricweb.z5.web.core.windows.net/oufr/7.28.1/#/examples/button

![image](https://user-images.githubusercontent.com/1434956/63889171-5121f200-c995-11e9-889d-a8d5f2df19b6.png)

Should be like this:

![image](https://user-images.githubusercontent.com/1434956/63889321-9e9e5f00-c995-11e9-897d-3404d1b8adb5.png)

**Comment by maxwellred** [comment]
@micahgodbolt Thanks, I'll look into the high contrast selectors now.

**Comment by micahgodbolt** [comment]
While you're looking at high contrast, check out the focused state of the primary button. It looks like the internal focus ring is getting covered up by the background.
